module/sample_linux.json

```diff
@@ -0,0 +1,44 @@
+{
+    "components": [
+      {
+        "model": "viam:lidar:rplidar",
+        "name": "rplidar",
+        "namespace": "rdk",
+        "type": "camera",
+        "depends_on": []
+      }
+    ],
+    "services": [
+      {
+        "name": "slam-test",
+        "model": "viam:slam:cartographer",
+        "type": "slam",
+        "attributes": {
+          "port": "localhost:8083",
+          "data_dir": "/Users/kat/viam/datasets/test-delete-me",
```

Comment:
[q] should we hardcode your name in the sample config? Why dont we do something like `HOME_DIRECTORY/DATA_DIR` to indicate to the user that they should use replace this value?

Reply:
updated